meteor/lib/collections/PeripheralDevices.ts

```diff
@@ -64,7 +64,17 @@ export enum PlayoutDeviceType { // to match DeviceType in TSR
 	PANASONIC_PTZ = 5,
 	// TCPSEND = 6, // to be implemented
 	HYPERDECK = 7,
-	PHAROS = 8
+	PHAROS = 8,
+	SCANNER = 9,
+	LAUNCHER = 10
+}
+export const DefaultPlayoutDevicePort = {
```

Comment:
This looks strange and has a non-typed dependency to PlayoutDeviceType.
It would be better to replace it with an enum, or perhaps a
```
function getDefaultPort (deviceType: PlayoutDeviceType) {
  switch(deviceType) {
    case PlayoutDeviceType.CASPARCG: return 5250
    [...]
  }
}
```